script.js

```diff
@@ -288,9 +265,12 @@ if (
   const question = document.createElement("p");
   // When user clicks on "book this room", run and create function to open modal
   openModal.addEventListener("click", function (e) {
-    if (e.target.classList.contains("modal-open"))
+    if (e.target.classList.contains("modal-open")) {
```

Comment:
What you found here is a textbook example of the types of bugs that will sometimes occur when using if statements without curly braces. The person who wrote this first may have meant to append `backDrop` and `addEventListener()` only when the `modal-open` class is set, but because of missing braces, only the append happens conditionally, whereas `addEventListener()` happens always.

This is why I always recommend against using if statements without braces, even if they are syntactically legal. I will admit that these bugs are rare, but they do happen, as you've experienced here.